### ISSUE TITLE
Why use fixed learning rate?

### ISSUE THREAD

**Issue by realwill** [yuanming-hu/fc4#29]
Is it better to use fixed learning rate? And why?

**Comment by yuanming-hu** [comment]
Thanks for your question. There's no specific reason - I didn't try decaying learning rate since that would add more hyperparameters to tune...

**Comment by realwill** [comment]
got it, thanks :)
